geoportal/geoportailv3_geoportal/static-ngeo/js/olcs/Lux3DManager.js

```diff
@@ -131,7 +131,7 @@ const exports = class extends ngeoOlcsManager {
   init3dTiles(visible) {
     const scene = this.ol3d.getCesiumScene();
     this.tiles3dLayers_.forEach((layer) => {
-      const url = this.tiles3dUrl_ + layer;
+      const url = this.tiles3dUrl_ + layer + "/tileset.json";
```

Comment:
you may have checked if `tileset.json` is present before adding it.

Reply:
Like making an HTTP requests to the `tileset.json` ?

Reply:
is present *in the url* (-: